packages/babel-preset-stage-1/README.md

```diff
@@ -2,6 +2,16 @@
 
 > Babel preset for stage 1 plugins.
 
+You can read more about the stages @ http://www.2ality.com/2015/11/tc39-process.html but the gist of stage 1 is:
```

Comment:
These get rendered on http://babeljs.io/docs/plugins/preset-stage-1/#install so we'd want to use the markdown link syntax for these.